src/plugins/mission/include/plugins/mission/mission.h

```diff
@@ -1,11 +1,11 @@
 #pragma once
 
+#include <cmath>
 #include <vector>
 #include <memory>
 #include <functional>
```

Comment:
Regarding the Windows error, does it require this?

```suggestion
#include <functional>
#include <string>
```